plugins/vault-login.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: vault-login
+spec:
+  version: v0.0.2
+  homepage: https://github.com/FalcoSuessgott/kubectl-vault-login
+  shortDescription: A kubectl plugin for authenticating to Kubernetes using HashiCorp Vault's Kubernetes Secret Engine
```

```suggestion
  shortDescription: Authenticate with Vault Kubernetes Secret Engine
```
We truncate at 55 chars